crates/zksync/compiler/src/libraries.rs

```diff
@@ -1,24 +1,34 @@
-//! Handles resolution and storage of missing libraries emitted by zksolc
+//! Handles resolution nd storage of missing libraries emitted by zksolc
 
 use std::{
     fs,
     io::Write,
     path::{Path, PathBuf},
 };
 
+use serde::{Deserialize, Serialize};
 use tracing::{trace, warn};
 
 use foundry_compilers::info::ContractInfo;
 
-use crate::ZkMissingLibrary;
+#[derive(Debug, Clone, Deserialize, Serialize)]
+/// Missing Library entry
```

Comment:
nit: docs are canonically before any derives

```suggestion
/// Missing Library entry
#[derive(Debug, Clone, Deserialize, Serialize)]
```